app/src/userpages/components/ShareSidebar/Sidebar.jsx

```diff
@@ -532,7 +529,12 @@ const ShareSidebar = connect(({ user }) => ({
         // load latest permissions
         // required to set base permissions to whatever changes were successful
         await propsRef.current.loadPermissions()
-        if (!isMounted()) { return }
+
+        if (!isMounted()) {
+            // close sidebar if permission loading failed
+            propsRef.current.onClose()
```

Comment:
This is bit hacky, if user removes his own permission, the request fails and the root component unmounts on null permissions and leaves the dialog open and empty.